src/components/Filters/Filters.tsx

```diff
@@ -43,15 +43,16 @@ const Filters: React.FC<FiltersProps> = ({
   isLoading = false,
 }) => {
   const [filtersValues, setFiltersValues] = useState<Array<Filter>>(state);
-  const [isRemoveDisabled, setRemoveDisabled] = useState(false);
+  const [isDefaultState, setIsDefaultState] = useState(true);
   const [hasUnappliedFilters, setHasUnappliedFilters] = useState(false);
 
   useEffect(() => {
     const defaultState = getDefaultState(fields);
+
     if (equals(filtersValues, defaultState)) {
```

Comment:
`setIsDefaultState(equals(filtersValues, defaultState))` instead of those 5 lines?

Reply:
Oh, sure! Refactored